<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddingNewWeatherRecord</class>
 <widget class="QWidget" name="AddingNewWeatherRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>382</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>382</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление записи</string>
  </property>
  <widget class="QPushButton" name="addRecordPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="weekdayLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>301</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Введите день недели</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>301</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Введите дату</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>301</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Введите месяц</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="labelLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>301</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Введите основные характеристики</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperatureLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>301</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Введите температуру</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="whichLineLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>301</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>В какое поле хотите добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="necessary">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="necessary_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="necessary_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="necessary_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="necessary_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
